BusinessLayer/SongControl/SongControl.h

```diff
@@ -14,7 +14,7 @@
 #include <QVector>
 #include <QFileInfo>
 #include <QDebug>
-
+#include <QDir>
```

Comment:
QDir isn't used in the header, it's better to put it in the `cpp` file to speed up compile times. Because header files are compiled every time they're included, while cpp files are only compiled once, so the simpler a `.h` file the better.